libsplinter/src/transport/socket/tcp.rs

```diff
@@ -55,7 +55,7 @@ impl Transport for TcpTransport {
                 FrameError::UnsupportedVersion => ConnectError::ProtocolError(
                     "Unable to connect; remote version is not with in range".into(),
                 ),
-                FrameError::IoError(err) => ConnectError::IoError(err),
+                FrameError::IoError(err) => ConnectError::from(err),
```

Comment:
Typo in commit message: s/ErrorKine/ErrorKind/

Reply:
fixed